docs/user/datastores/index_config.rst

```diff
@@ -559,6 +559,50 @@ Full Example
     sft.getUserData().put("table.splitter.options",
         "id.pattern:[0-9a-f],attr.name.pattern:[a-z],z3.min:2018-01-01,z3.max:2018-01-31,z3.bits:2,z2.bits:4");
 
+Configuring Query Interceptors
+------------------------------
+
+GeoMesa provides a chance for custom logic to be applied to a query before executing it. Query interceptors must
+be specified through user data in the simple feature type, and may be set before calling ``createSchema``, or
+updated by calling ``updateSchema``. To indicate query interceptors, use the key ``geomesa.query.interceptors``:
+
+.. code-block:: java
+
+    sft.getUserData().put("geomesa.query.interceptors", "com.example.MyQueryInterceptor");
```

Comment:
Since this key is plural... is it possible to set multiple interceptors?!

Reply:
yep! comma-separated

Reply:
see next line haha :)

Reply:
Dang!  I was trying to maintain the illusion that I read these PRs...

Reply:
Ok, Is the order of operations clear?

Reply:
ah, possibly not in the docs - should it be clarified? they are executed in the order they are declared.

Reply:
added a note